lib/matplotlib/widgets.py

```diff
@@ -380,11 +381,29 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
             The color of the line at the *valinit* position. Set to ``'none'``
             for no line.
 
+        track_color : color, default: 'lightgrey'
+            The color of the background track. The track is accessible for
+            further styling via the *track* attribute.
+
+        handle_style : dict
+            Properties of the slider handle. Default values are
+
+            ========= ===== ======= ========================================
+            Key       Value Default Description
+            ========= ===== ======= ========================================
+            facecolor color 'white' The facecolor of the slider handle.
+            edgecolor color '.75'   The edgecolor of the slider handle.
+            size      int   10      The size of the slider handle in points.
+            ========= ===== ======= ========================================
+            Other values will be transformed as marker{foo} and passed to the
```

Comment:
```suggestion

            Other values will be transformed as marker{foo} and passed to the
```
ReST requires a blank line after a table.

Reply:
Thanks! I applied these in the rebase